assets/airports/kric.json

```diff
@@ -0,0 +1,9359 @@
+        "V38": ["GVE", "EMMIE", "ROOKY", "RIC", "NEMAN", "HCM", "JUDLU", "TERKS", "CCV"],
+        "V155": ["RDU", "WIPER", "BEXGO", "LVL"],
+        "V157": ["LVL", "MANGE", "DALTO", "MATOX", "RIC", "SVILL", "TAPPA", "DUPEY", "COLIN", "PXT"],
+        "V260": ["FKN", "WAIKS", "HPW", "RIC", "FAK"]
+    },
+    "sids": {
+        "COLIN8": {
+            "icao": "COLIN8",
+            "name": "Colin Eight",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], "_RIC05010"],
+                "KRIC16": [["^_RW161DME", "A007+"], "_RIC12107", "_RIC07410", "MODEL"],
+                "KRIC20": [["^_RW201DME", "A007+"], "_RIC12107", "_RIC07410", "MODEL"],
+                "KRIC34": [["^_RW341DME", "A007+"], "_RIC05010"]
+            },
+            "body": ["COLIN"],
+            "exitPoints": {
+                "COLIN": []
+            },
+            "draw": [
+                ["_RW021DME", "_RIC05010", "_RW341DME"],
+                ["_RIC05010", "COLIN*"],
+                ["_RW161DME", "_RIC12107", "_RW201DME"],
+                ["_RIC12107", "_RIC07410", "MODEL", "COLIN"]
+            ]
+        },
+        "KALLI7": {
+            "icao": "KALLI7",
+            "name": "Kalli Seven",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SUGGR", "A50+"]],
+                "KRIC16": [["^_RW161DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC20": [["^_RW201DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC34": [["^_RW341DME", "A007+"], ["SUGGR", "A50+"]]
+            },
+            "body": ["KALLI"],
+            "exitPoints": {
+                "DRAIK": ["READE", "DRAIK"],
+                "READE": ["READE"],
+                "SANNY": ["READE", "DRAIK", "SANNY"]
+            },
+            "draw": [
+                ["_RW021DME", "SUGGR", "_RW341DME"],
+                ["SUGGR", "KALLI", "READE*", "DRAIK*", "SANNY*"],
+                ["_RW161DME", "ITTEM", "_RW201DME"],
+                ["ITTEM", "KALLI"]
+            ]
+        },
+        "LUCYL6": {
+            "icao": "LUCYL6",
+            "name": "Lucyl Six",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"],
+                "KRIC16": [["^_RW161DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
+                "KRIC20": [["^_RW201DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
+                "KRIC34": [["^_RW341DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"]
+            },
+            "body": ["LUCYL"],
+            "exitPoints": {
+                "LUCYL": []
+            },
+            "draw": [
+                ["_RW021DME", "SJAYY", "_RW341DME"],
+                ["SJAYY", "KAMMI", "LUCYL*", "HAYYU", "BUNKK", "KEGME"],
+                ["_RW161DME", "KEGME", "_RW201DME"]
+            ]
+        },
+        "READE7": {
+            "icao": "READE7",
+            "name": "Reade Seven",
+            "rwy": {
```

Comment:
Less noticeable than with the COLIN8, but there is still a problem with magnetic deviation by the looks of it
![image](https://user-images.githubusercontent.com/13125311/152981915-15bbd8cf-6572-4ced-b9c3-cb9988bcc80f.png)